src/flypad-backend/src/Aircraft/AircraftProcedures.h

```diff
@@ -1,10 +1,12 @@
 #pragma once
 
-#include <map>
-#include <string>
 #include <vector>
-
-using namespace std;
+#include <algorithm>
+#ifdef DEBUG
+#include <iostream>
+#endif
+#include <array>
+#include <string>
```

Comment:
```suggestion
#include <string>
#ifdef DEBUG
#include <iostream>
#endif
```

Remove array - not needed any more, right?